src/bmark/run_solver.jl

```diff
@@ -3,9 +3,11 @@ export display_header, solve_problems, solve_problem
 type SkipException <: Exception
 end
 
+const uncstats = [:obj, :dual_feas, :neval_obj, :neval_grad, :neval_hprod, :iter, :elapsed_time, :status]
```

Comment:
I was thinking this could an option to pass to `solve_problem` so that the printing is more personalized. However, I do see this getting out of hand.
I can
- Include a `constats` with the other options and add an option to `solve_problem` to pass personalized columns; or
- Remove `uncstats` and use a general line.